### DISCUSSION TITLE
New video format

### DISCUSSION THREAD

**Discussion by TPsqr** [olivierfriard/BORIS#546] [Ideas]
Hello,

As part of our reseach on pig behaviour, we're using RF video files (RealFlash Clip) which are highly compressed files. 
It seems that BORIS is not recognizing it as a media file. Could you make it possible to use this format with BORIS ?
Thanks

**Comment by olivierfriard** [comment]
Hello,
try to re-encode / resize your video.
See Tools > Re-encode/ resize
